src/components/TextInput/styles.tsx

```diff
@@ -0,0 +1,41 @@
+import styled from 'styled-components';
+import { Inter } from 'next/font/google';
+import COLORS from '../../styles/colors';
+
+const inter = Inter({ subsets: ['latin'] });
+
+export const InputText = styled.input<{ $error: boolean }>`
+  color: ${COLORS.existingInputColor};
+  font-size: 1rem; // 16px
+  font-family: ${inter.style};
```

Comment:
`inter.style` by itself looks like `font-family: ...` already, no need to add `font-family:` before it.

Reply:
I'm a little confused about what you means but i'll look into it!

Reply:
I meant that if you just inserted `${inter.style}` like that without `font-family: ` before it, it would still work fine, since `inter.style` already includes the property name.